### ISSUE TITLE
NSFW filter toggle [FEATURE REQUEST]

### ISSUE THREAD

**Issue by MontyCristo** [hlky/stable-diffusion-webui#160]
Could you please add a toggle to turn on the NSFW filter? You have the best GUI I've seen and I would really love to use it if there was a config option to turn that on.

**Comment by hlky** [comment]
NSFW filter is on `diffusers` codebase. will have to check if it can be easily ported

**Comment by altryne** [comment]
@MontyCristo you want to ENABLE that toggle to get blurred results? 
hmmm maybe if this UI is going to get used in collab and save images to google drive that's not a bad idea 🤔

@hlky could this be helpful? 
https://github.com/CompVis/stable-diffusion/blob/main/scripts/txt2img.py#L87-L94

**Comment by MontyCristo** [comment]
@altryne Yes, that's right. I would really like to ENABLE the blurring or blacking filter. I'd love to be using this tech for some family friendly uses.

**Comment by SweetSophia** [comment]
> @altryne Yes, that's right. I would really like to ENABLE the blurring or blacking filter. I'd love to be using this tech for some family friendly uses.

You could use prompts to enable some kind of nsfw filter. You can exclude certain stuff from images.
